src/beacon/use-beacon-handler.hook.ts

```diff
@@ -14,11 +12,5 @@ export const useBeaconHandler = () => {
     BeaconHandler.init(message =>
```

Comment:
short return

Reply:
typescript yells an error `Type 'Promise<void>' is not assignable to type 'void | Destructor'.`

Reply:
add `void` before `BeaconHandler`